pkg/bitwarden/fake.go

```diff
@@ -50,6 +53,106 @@ func (c fakeClient) GetPassword(itemName string) ([]byte, error) {
 	return nil, fmt.Errorf("failed to find password in item %s", itemName)
 }
 
+func getNewUUID() (string, error) {
```

Comment:
probably irrelevant here, but cryptographically secure entropy tends to be limited and isn't needed here so something like `time.Now().Nanoseconds()` should be enough

Reply:
If we were to use nanoseconds and have tests running in parallel, we "might" have both of them executing at the same time resulting in the same ID and flaky tests?
Also what do you mean by limited here?

Reply:
Chances of collissions are so slim that this is part of the UUID1 standard, whose sole purpose is to create unique (but not cryptographically secure) uids: https://stackoverflow.com/questions/40411451/why-will-timeuuid-not-have-any-collisions.

See https://en.wikipedia.org/wiki/Entropy_(computing)#Practical_implications for an explanation of why it can be limited

Reply:
So by limiting, you meant that there is a chance that the system would get stuck while waiting for random characters to appear

Reply:
Yes, either for this application or some other, as the entropy pool is per host. There is no reason to use cryptographically secure entropy when the only thing you need is something that is collision-free.